spec/configuration_spec.rb

```diff
@@ -7,26 +7,28 @@
 RSpec.describe Readyset::Configuration do
   let(:config) { described_class.new }
 
-  context 'when no database_url is specified' do
-    it 'defaults to dummy url' do
-      default_url = 'postgres://user:password@localhost:5432/readyset'
-      expect(config.database_url).to eq default_url
+  describe 'database_url' do
```

Comment:
I think this change will go away once we rebase this branch on top of main now that #59 has been merged